packages/models/proto/v2/eservice/eservice.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package eservice.v2;
+
+message EServiceV2 {
+  string id = 1;
+  string producerId = 2;
+  string name = 3;
+  string description = 4;
+  EServiceTechnologyV2 technology = 5;
+  optional EServiceAttributesV2 attributes = 6;
+  repeated EServiceDescriptorV2 descriptors = 7;
+  optional int64 createdAt = 8;
+}
+
+message EServiceAttributeValueV2 {
+  string id = 1;
+  bool explicitAttributeVerification = 2;
+}
+
+message EServiceAttributeV2 {
+  optional EServiceAttributeValueV2 single = 1;
+  repeated EServiceAttributeValueV2 group = 2;
+}
+
+message EServiceAttributesV2 {
+  repeated EServiceAttributeV2 certified = 1;
+  repeated EServiceAttributeV2 declared = 2;
+  repeated EServiceAttributeV2 verified = 3;
+}
+
+message EServiceDescriptorV2 {
+  string id = 1;
+  string version = 2;
+  optional string description = 3;
+  repeated EServiceDocumentV2 docs = 4;
+  EServiceDescriptorStateV2 state = 5;
+  optional EServiceDocumentV2 interface = 6;
+  repeated string audience = 7;
+  int32 voucherLifespan = 8;
+  int32 dailyCallsPerConsumer = 9;
+  int32 dailyCallsTotal = 10;
+  optional AgreementApprovalPolicyV2 agreementApprovalPolicy = 11;
```

Comment:
```suggestion
  AgreementApprovalPolicyV2 agreementApprovalPolicy = 11;
```

Reply:
To reuse the same readmodel I have to choose a default value in case of undefined (the readmodel that I have to place inside the event could have this field `undefined`)

Reply:
I see your point, in the [as-is version](https://github.com/pagopa/interop-be-catalog-management/blob/1.0.x/models/src/main/scala/it/pagopa/interop/catalogmanagement/model/PersistentAgreementApprovalPolicy.scala#L7) we use `Automatic` as default value